src/main/java/org/apache/xml/security/keys/OriginatorKeyInfo.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.xml.security.keys;
```

Comment:
I think this should be in the `org.apache.xml.encryption` package with the other xml encryption specific keyinfo types such as `EncryptedKey`. I think in general it is best if the encryption and signature packages and structures are kept separate, as this provides a cleaner separation and also more easily allows someone to omit xml encryption if it isn't needed.

Same comment for `RecipientKeyInfo`.

Reply:
Done all "encryption specific keys" are now moved to org.apache.xml.encryption.keys...